lib/features/family/features/reflect/bloc/summary_cubit.dart

```diff
@@ -77,9 +81,40 @@ class SummaryCubit extends CommonCubit<SummaryDetails, dynamic> {
     );
   }
 
-  Future<int> getTotalGamePlays() async =>
-      _reflectAndShareRepository.getTotalGamePlays();
+  Future<void> doneButtonPressed() async {
+    final gameplays = await _reflectAndShareRepository.getTotalGamePlays();
+    final playsBeforePopup =
+        _reflectAndShareRepository.getStoreReviewMinimumalGameCount();
 
-  Future<int> getStoreReviewGameCount() async =>
-      _reflectAndShareRepository.getStoreReviewGameCount();
+    if (await InAppReview.instance.isAvailable() &&
+        gameplays == (playsBeforePopup - 1)) {
+      await AnalyticsHelper.logEvent(
+        eventName: AmplitudeEvents.inAppReviewTriggered,
+      );
+
+      await InAppReview.instance.requestReview();
+    }
+
+    if (_audioPath.isNotEmpty) {
+      await sendAudioAndNavigate(_audioPath);
```

Comment:
Can be simplified. Remove sendAudioAndNavigate method, just do await shareAudio here and remove the return.